error-establishing-connection-to-a-data-link-server.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Error Establishing a Connection to a {{site.dls}}
+description: Check the server address format if you receive the &quot;Error accessing Data Link Server at http&#58;//localhost&#58;9090&#58; connect ECONNREFUSED&quot; error.
+type: troubleshooting
+page_title: Unsuccessful connection to {{site.dls}}
+slug: error-establishing-connection-to-a-data-link-server
+position: 
+tags: data connector, data link server, data, sql, everlive, backend, connection
+teampulseid: 
+ticketid: 981179, 967275, 950185
+pitsid:
+---
+
+## Environment
+
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Data</td>	
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Data Connectors</td>	
+  </tr>
+</table>
+
+## Description
+
+When configuring a {{site.dc}} to a database that requires the {{site.dls}}, an error occurs after clicking the **Test** button.
+
+At the same time, opening the {{site.dls}} URL in a web browser returns *Result: OK*.  
+
+## Error Message
+
+`Error accessing Data Link Server at http://localhost:9090: connect ECONNREFUSED.`
+
+## Cause
+
+The cause for the error is that {{site.tp}} is not able to resolve the provided {{site.dls}} URL because *localhost* is not a valid address. You must provide a public FQDN or IP address to {{site.tp}}.
```

Comment:
... fully qualified domain name (FQDN) ...

When using abbreviations, write the full name first.